Jenkinsfile

```diff
@@ -1,26 +1,45 @@
 pipeline {
     agent { label 'mobile-builder-ios-pull-request'  }
-    environment {
-       PRODUCT = 'couchbase-lite-ios-ee'
-   }
     stages {
         stage('Checkout'){
             steps {
                 sh """
-                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git
-                    pushd ${env.PRODUCT}
+                    #!/bin/bash
+                    set -e
+                    shopt -s extglob dotglob
+
+		    # move PR related repo to tmp folder
+		    mkdir tmp
+                    mv !(tmp) tmp
+
+		    # clone and update submodules here
+                    git clone https://github.com/couchbaselabs/couchbase-lite-ios-ee.git --branch $CHANGE_TARGET
+		    # submodule update inside lite-ios
+		    pushd couchbase-lite-ios-ee
                     git submodule update --init --recursive
-                    ./Scripts/prepare_project.sh
-                    cd couchbase-lite-ios
-		    git checkout master
-		    git pull origin master
-                    popd
+		    popd
+
+		    # restructure folders
+		    mv couchbase-lite-ios-ee/* .
+		    rm -rf couchbase-lite-ios && mv tmp couchbase-lite-ios
+		    
+		    # submodule update inside lite-ios
+		    pushd couchbase-lite-ios
+                    git submodule update --init --recursive
+		    popd
+
+		    # remove tmp folders
+		    rm -rf tmp/*
```

Comment:
tmp has already been renamed to couchbase-lite-ios in line 24.  The rest is good.